### ISSUE TITLE
Bring back signleton

### ISSUE THREAD

**Issue by mitinarseny** [rethinkdb/rethinkdb-python#112]
Before it was easy just to write
```python
import rethinkdb as r

conn = r.connect(...)
r.table(...).run(conn)
```
But documentation (simply README) of rethinkdb python driver >=2.4.0 insist that now the proper way to do it is this:
```python
from rethinkdb import RethinkDB

r = RethinkDB()
conn = r.connect(...)
r.table(...).run(conn)
```
It's OK to do it in single function. But it is NOT OK to do it in multiple functions and pass around the `rethinkdb_instance` parameter in each function that makes requests to your rethinkdb.

**Comment by gabor-boros** [comment]
Hello @mitinarseny, maybe I misunderstood something, but this is why we created a “shortcut” for this problem.

> Although we recommend to use the import used in the examples, to help the migration from rethinkdb<2.4 we introduced a shortcut which can easily replace the old “import rethinkdb as r” import with “from rethinkdb import r”.

**Comment by mitinarseny** [comment]
@gabor-boros 
Could you explain about shortcut, please? 

Let me explain you more about the problem that I faced.
I have just encountered a problem: I updated `rethinkdb` dependency of my project to `2.4.1`. Here is an example of code that now doesn't work:
```python
import os
import rethinkdb as r

RDB_CONFIG = {
    'host': os.environ['RDB_HOST'],
    'port': os.environ['RDB_PORT'],
    'db': os.environ['RDB_DB'],
    'user': os.environ['RDB_USER'],
    'pass': os.environ['RDB_PASS'],
}


def db_connection() -> r.Connection:
    return r.connect(
        host=RDB_CONFIG['host'],
        port=RDB_CONFIG['port'],
        db=RDB_CONFIG['db'],
        user=RDB_CONFIG['user'],
        password=RDB_CONFIG['pass']
    )
```
I use `db_connection()` like this:
```python
async def some_function():
    async with db_connection() as conn:
        # do some stuff with conn
```
Here is the error that I get when run my code:
```
Traceback (most recent call last):
  ...
  File ".../db.py", line 14, in <module>
    def db_connection() -> r.Connection:
AttributeError: module 'rethinkdb' has no attribute 'Connection'
```
There was no such error with previous versions of rethinkdb (e.g. `2.3.0post6`)
I guess this is connected with having to do something like this:
```python
from rethinkdb import RethinkDB

r = RethinkDB()
# now work with r
```
But how can I implement this without doing something like this?
```python
# file rethinkdb_wrapper.py
from rethinkdb import RethinkDB

r = RethinkDB()
```
```python
# file db.py
import os

from rethinkdb_wrapper import r

RDB_CONFIG = {
    'host': os.environ['RDB_HOST'],
    'port': os.environ['RDB_PORT'],
    'db': os.environ['RDB_DB'],
    'user': os.environ['RDB_USER'],
    'pass': os.environ['RDB_PASS'],
}


def db_connection() -> r.Connection:
    return r.connect(
        host=RDB_CONFIG['host'],
        port=RDB_CONFIG['port'],
        db=RDB_CONFIG['db'],
        user=RDB_CONFIG['user'],
        password=RDB_CONFIG['pass']
    )
```
It doesn't look like good solution, does it?

**Comment by gabor-boros** [comment]
@mitinarseny With the version 2.4.2 we will implement a shortcut for exactly those cases what you have to help the migration from 2.3.x to 2.4.x. This shortcut is implemented here https://github.com/rethinkdb/rethinkdb-python/blob/master/rethinkdb/__init__.py#L88.

So after the release of 2.4.2 the only think what you should change is the import of rethinkdb. So, instead of `import rethinkdb as r` you will need to write `from rethinkdb import r`.

The release will happen, when PR #106, #114 are merged and issue #83, #104 and #110 are resolved.

**Comment by gabor-boros** [comment]
V2.4.2 is out
